src/renderer/features/governance-operation-details/governance-operation-details-model.tsx

```diff
@@ -12,13 +12,37 @@ export const governanceOperationDetailFeature = createFeature({
 });
 
 governanceOperationDetailFeature.inject(multisigOperationsFeature.slots.operationDetails, {
-  render: ({ operation }) => <GovernanceVoteDetails operation={operation} />,
+  render: ({ operation }) => {
+    const transaction = getTransactionFromMultisigTx(operation);
+
+    if (
+      transaction?.type &&
+      [TransactionType.UNLOCK, TransactionType.VOTE, TransactionType.REVOTE, TransactionType.REMOVE_VOTE].includes(
+        transaction.type,
+      )
+    ) {
+      return <GovernanceVoteDetails operation={operation} />;
+    }
```

Comment:
In case when we check tx, maybe delete duplication from GovernanceVoteDetails component?